docs/faq/onboarding/lukso-standards.md

```diff
@@ -0,0 +1,70 @@
+
+### Why will LUKSO have an advantage in LSP adoption?
+
+LUKSO has quite an advantage on first implementations, as the whole user base will be **directly onboarded through Universal Profiles**, setting the perfect environment for LSPs without creating sub-sections. Furthermore, LUKSO can subsidize early adopters on the new chain until a substantial user base is reached. Other networks must update to the new behaviors instead, as they have already onboarded people with other tools or standards.
+
+### How many pictures or datasets can be added to NFTs 2.0?
+
+Thanks to the LSP4-DigitalAsset-Metadata standard that is used within [LSP7](https://docs.lukso.tech/standards/nft-2.0/LSP7-Digital-Asset) Token and [LSP8](https://docs.lukso.tech/standards/nft-2.0/LSP8-Identifiable-Digital-Asset) NFT, an asset can hold unlimited metadata, which could store or link to multiple images, videos, gifs, and texts if needed. These datasets could then also be updated if configured.
+
+### When using the LSP9 Vault, does the NFT store on-chain?
+
+Yes. The vault is a subcontract of the owned UP smart contract account on-chain, with its contracts sent to them. While the NFT contract is stored on-chain, most of its metadata and contents will come from the **storage solution link** you specified in the **JSON schema**. The link could point to any data hub, home environment, or decentralized storage application.
+
+### Do LSPs define the storage solution of the contract’s data?
+
+No, the LSPs do not specify the type of storage solution. Any content link can be attached to the contract. This allows assets and profiles to be stored across various existing and upcoming data solutions. The link can be specified in the JSON schema.
+
+### What is the difference between ERC721, ERC1155, LSP7, and LSP8?
+
+[ERC721](https://eips.ethereum.org/EIPS/eip-721) and [ERC1155](https://eips.ethereum.org/EIPS/eip-1155) are widely used Non-Fungible Token (NFT) standards on Ethereum. While they sent off the NFT adoption and were the first of their kind, they have certain limitations. By default, the standards lack metadata for the asset, limit the NFT's storage and attachments, and restrict interaction capabilities due to not being updatable and only having simple EOA management.
+
+LUKSO's NFT standards, namely [LSP7](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-7-DigitalAsset.md) and [LSP8](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md), are designed to overcome these limitations. The standards enhance the interaction potential. For instance, when an asset is transferred, the sender and recipient are notified. You can also attach multiple owners, add various media content, and update its content over time.
```

Comment:
I recommend to link to docs page here instead of GitHub pages.

https://docs.lukso.tech/standards/nft-2.0/LSP7-Digital-Asset

https://docs.lukso.tech/standards/nft-2.0/LSP8-Identifiable-Digital-Asset

Use relative links pls